io-reactor/src/reactor.rs

```diff
@@ -205,6 +205,7 @@ impl<S: Handler> Controller<S> {
 fn reset_fd(fd: &impl AsRawFd) -> io::Result<()> {
     let mut buf = [0u8; 4096];
 
+    // cloudhead: Why is this not calling popol? This is different for all wakers.
```

Comment:
I do not entirely understand what do you mean at all. I took this function from nakamoto and it doesn't use popol; it is a reset to the waker called from a single place (when waker queue is filled up I assume)

Reply:
Ah, well this function is actually from popol. The waker is usually implementation dependent, and this is how you reset the popol waker. Other wakers will be reset with different code. In any case, I would call popol directly here: https://github.com/cloudhead/popol/blob/master/src/lib.rs#L476